autoload/fzf/vim.vim

```diff
@@ -324,6 +324,18 @@ function! fzf#vim#history(...)
 endfunction
 
 " ------------------------------------------------------------------
+" GitFiles
+" ------------------------------------------------------------------
+
+function! fzf#vim#gitfiles(...)
+  call s:fzf({
+  \ 'source':  'git ls-tree --name-only --full-tree -r HEAD',
```

Comment:
I don't think we should use `--full-tree`. Vim will unable to find the file if the current working directory is not the root of the repo.